packages/connect-mobile/src/index.ts

```diff
@@ -47,7 +47,7 @@ export class TrezorConnectDeeplink implements ConnectFactoryDependencies<Connect
 
     private validateConnectSrc(connectSrc?: string) {
         if (!connectSrc) return DEFAULT_DOMAIN_MAJOR_VER;
-        if (connectSrc === 'trezorsuitelite://connect') return connectSrc;
+        if (connectSrc === 'trezorsuite://connect') return connectSrc;
```

Comment:
hmm do we really need to change this? isn't it kind of internal stuff? If versions (3rd party and trezor-suite-app) are not updated in sync (and they won't), this change will likely break majority of 3rd party implementations. Or am I missing something? cc @martykan

Reply:
Don't worry, this is only for local dev, we use the HTTPS links in production for better security